pkg/siteconfig/clusterinstance_test.go

```diff
@@ -21,6 +28,62 @@ var testSchemes = []clients.SchemeAttacher{
 	siteconfigv1alpha1.AddToScheme,
 }
 
+func TestNewClusterInstanceBuilder(t *testing.T) {
+	generateCIBuilder := NewCIBuilder
```

Comment:
You can just use `NewCIBuilder` directly in the test instead of creating the `generateCIBuilder` variable